src/ntfsck.c

```diff
@@ -4413,12 +4458,12 @@ int main(int argc, char **argv)
 
 	ntfs_log_set_levels(NTFS_LOG_LEVEL_INFO);
 	ntfs_log_clear_levels(NTFS_LOG_LEVEL_TRACE|NTFS_LOG_LEVEL_ENTER|NTFS_LOG_LEVEL_LEAVE);
-
+	pb_flags = NTFS_PROGBAR;
```

Comment:
where is this used ?

Reply:
if someone who do not want print 'progress bar', he can use option '-q'.
it set pb_flag |= ~NTFS_PROGBAR, then it set prgress_init() with no progress bar option.